### DISCUSSION TITLE
phonon dispersion unfolding

### DISCUSSION THREAD

**Discussion by Lanlin123** [SSCHAcode/python-sscha#93] [Q&A] [answered]
Dear SSCHA developers,

    I doubt whether the phonon dispersion calculated from supercell by SSCHA needs to be unfolded to unitcell to compare with the experiment data.

Best wishes,
Lily Du.

**Comment by mesonepigreco** [comment, marked as answer]
Hi Lily Du,
Inside the CellConstructor directory, there is a script to plot the phonon dispersion. 
You can edit the variables at the beginning to point toward your files and define your own path.

https://github.com/SSCHAcode/CellConstructor/blob/master/tutorials/PlotPhononDispersion/plot_dispersion.py


If you are using the development version of both CellConstructor and python-sscha (the latest commit on the master), there is also a tutorial on Gold that employs new functions to do the same. These functions will be released in the 1.2 version very soon:

https://github.com/SSCHAcode/python-sscha/blob/master/python-sscha.pdf
(Section 3.2)

Hope this answer your question.
Best regards,
Lorenzo

**Reply by Lanlin123** [reply]
Hi Lorenzo,

    Thank you a lot ! I wonder if the supercell definition in SSCHA be equal to the q mesh definition, i.e. to set a 4×4×4 supercell ,we only need set a 4×4×4 q mesh using unit cell. The tutorials give me this sence, and I don't know whether it's true.

Best, 
Lily

**Reply by mesonepigreco** [reply]
Yes, this is true, the dimension of the uniform q mesh and the supercell are exactly the same.
For example, if you have a supercell big 4 times a primitive vector along a direction, this is equivalent to computing 4 q points in reciprocal space (0, 1/4, 2/4, and 3/4 of the corresponding reciprocal vector).

So a 4x4x4 q uniform q mesh corresponds to a 4x4x4 supercell.

**Reply by Lanlin123** [reply]
Ok, I see. Thank you again.
